include/constants/hold_effects.h

```diff
@@ -161,6 +161,8 @@
 #define HOLD_EFFECT_LOADED_DICE         180
 #define HOLD_EFFECT_BOOSTER_ENERGY      181 // Not implemented.
 
+#define HOLD_EFFECT_BERSERK_GENE        182
```

Comment:
Move this below the other hold effects.

Reply:
should this be added to pre gen4 hold effects. if so keep the defined value or adjust all following?
otherwise this would be added to gen9 hold effects

Reply:
Oh, true. Then add
```c
// Gen8 hold effects
```

Reply:
Oh, true. Then add
```c
// Gen2 hold effect
```